jerry-core/ecma/base/ecma-gc.c

```diff
@@ -88,88 +88,45 @@ static void ecma_gc_mark (ecma_object_t *object_p);
 static void ecma_gc_sweep (ecma_object_t *object_p);
 
 /**
- * Get GC reference counter of the object.
- */
-static uint32_t
-ecma_gc_get_object_refs (ecma_object_t *object_p) /**< object */
-{
-  JERRY_ASSERT (object_p != NULL);
-
-  return (uint32_t) JRT_EXTRACT_BIT_FIELD (uint64_t, object_p->container,
-                                           ECMA_OBJECT_GC_REFS_POS,
-                                           ECMA_OBJECT_GC_REFS_WIDTH);
-} /* ecma_gc_get_object_refs */
-
-/**
- * Set GC reference counter of the object.
- */
-static void
-ecma_gc_set_object_refs (ecma_object_t *object_p, /**< object */
-                         uint32_t refs) /**< new reference counter */
-{
-  JERRY_ASSERT (object_p != NULL);
-
-  object_p->container = JRT_SET_BIT_FIELD_VALUE (uint64_t, object_p->container,
-                                                 refs,
-                                                 ECMA_OBJECT_GC_REFS_POS,
-                                                 ECMA_OBJECT_GC_REFS_WIDTH);
-} /* ecma_gc_set_object_refs */
-
-/**
  * Get next object in list of objects with same generation.
  */
-static ecma_object_t *
+static inline ecma_object_t *
```

Comment:
Are these (`ecma_gc_{get,set}_object_next`) functions still needed? With the proposed changes, they are merely wrappers around `ECMA_{GET,SET}_POINTER`. (`ecma_gc_{get,set}_object_refs` have been dropped as well, not caring about the asserts. So, do we need these?)